### ISSUE TITLE
@typescript-eslint/type-utils@npm:6.11.0: No candidates found

### ISSUE THREAD

**Issue by The-Code-Monkey** [typescript-eslint/typescript-eslint#7945]
### Before You File a Bug Report Please Confirm You Have Done The Following...

- [X] I have tried restarting my IDE and the issue persists.
- [X] I have updated to the latest version of the packages.
- [X] I have [searched for related issues](https://github.com/typescript-eslint/typescript-eslint/issues?q=is%3Aissue+label%3Abug) and found none that matched my issue.
- [X] I have [read the FAQ](https://typescript-eslint.io/linting/troubleshooting) and my problem is not listed.

### Relevant Package

utils

### Repro Code

```typescript
installing a package and this caused the package not to install
```

### Expected Result

should have installed

### Actual Result

@typescript-eslint/type-utils@npm:6.11.0: No candidates found

### Versions

yarn v4 and node 18 both latest

**Comment by bradzacher** [comment]
https://www.npmjs.com/package/@typescript-eslint/type-utils/v/6.11.0

Version is published and visible on NPM.
I updated the codebase at work to this version no problems this week.

So there's a problem with your setup or connection to npm.
